PROBot/BattleAI.cs

```diff
@@ -165,7 +163,7 @@ private bool RepeatAttack()
         {
             if (ActivePokemon.CurrentHealth > 0 && _client.ActiveBattle.RepeatAttack)
             {
-                _client.UseAttack(_lastAttackId);
+                _client.UseAttack(1);

```

Comment:
Are we sure the game client sends 1 to repeat the attack?
I thought it was the previous attack ID.

We can remove `_lastAttackId` if this is always 1.

Reply:
Yeah, I looked at PRO's code and it just sends 1.